src/index.html

```diff
@@ -1105,16 +1115,14 @@ <h3 class="heading-title">Why Maturify? </h3>
                 <div class="row">
                     <div class="col-lg-12 col-md-12 col-sm-12 col-xs-12">
                         <span>
-                            Copyright © 2017/2018 <a href="index.html" class="sub-footer__link">Maturify.</a>
+                            Copyright © 2020 <a href="index.html" class="sub-footer__link">Maturify.</a>
```

Comment:
Might be better if we can add the current year instead through a code of js like this: <script>document.write(new Date().getFullYear())</script>

Then we dont need to update each year.